src/main/java/com/glencoesoftware/omero/ms/image/region/ImageRegionMicroserviceVerticle.java

```diff
@@ -398,6 +405,13 @@ private void renderImageRegion(RoutingContext event) {
             response.setStatusCode(400).end(e.getMessage());
             return;
         }
+        if (imageRegionCtx.channels.size() > MAX_CHANNELS) {
```

Comment:
Need to check active channels. This array has all channels present. Should be all the channels > 0 as disabled ones are sent to the server as negative values. Should check this assumption with @knabar though.

Reply:
Yes, it still sends all channels, but only because that's the way it's always been done...